io/zenoh-transport/tests/unicast_authenticator.rs

```diff
@@ -606,6 +606,18 @@ fn authenticator_udp() {
     task::block_on(run(&endpoint));
 }
 
+#[cfg(feature = "transport_shm")]
+#[test]
```

Comment:
I believe there is no need to run this test every time on the CI (it will make increase its duration).
Mark the test as follows:
```
#[test]
#[ignore]
```
It can be then run with `cargo test -- --ignored`.